cli/cmd/weight/weight.go

```diff
@@ -71,58 +81,44 @@ func scaleAndAllocate(ctx *clicontext.CLIContext, args []string, pause bool, inc
 			service.Spec.RolloutConfig = &riov1.RolloutConfig{
 				Pause:           pause,
 				Increment:       increment,
-				IntervalSeconds: int(interval / time.Second),
+				IntervalSeconds: interval,
 			}
 			return nil
 		})
 		errs = append(errs, err)
-		reminder -= scale
-	}
-
-	// grab all services that already had weight allocated
-	var toAllocate []riov1.Service
-	total := 0
-	for _, s := range svcs {
-		// Don't count ones specified in the weight cmd
-		if _, ok := cmdSet[s.Name]; ok {
-			continue
-		}
-		if s.Status.ComputedWeight != nil && *s.Status.ComputedWeight > 0 {
-			total += *s.Status.ComputedWeight
-			toAllocate = append(toAllocate, s)
-		}
+		//reminder -= scale
 	}
 
 	// now allocate any remaining weight across those pre-weighted services
-	added := 0
-	for i, rev := range toAllocate {
-		resource, err := ctx.ByID(rev.Name)
-		if err != nil {
-			return err
-		}
-		weight := 0
-		if i == len(toAllocate)-1 {
-			weight = reminder - added
-		} else {
-			weight = int(float64(*rev.Status.ComputedWeight) / float64(total) * float64(reminder))
-			added += weight
-		}
-		err = ctx.UpdateResource(resource, func(obj runtime.Object) error {
-			s := obj.(*riov1.Service)
-			if s.Spec.Weight == nil {
-				s.Spec.Weight = new(int)
-			}
-			*s.Spec.Weight = weight
-			s.Spec.RolloutConfig = &riov1.RolloutConfig{
-				Pause:           pause,
-				Increment:       increment,
-				IntervalSeconds: int(interval / time.Second),
-			}
-			return nil
-		})
-
-		errs = append(errs, err)
-	}
+	//added := 0
+	//for i, rev := range toAllocate {
+	//	resource, err := ctx.ByID(rev.Name)
+	//	if err != nil {
+	//		return err
+	//	}
+	//	weight := 0
+	//	if i == len(toAllocate)-1 {
+	//		weight = reminder - added
+	//	} else {
+	//		weight = int(float64(*rev.Status.ComputedWeight) / float64(total) * float64(reminder))
+	//		added += weight
+	//	}
+	//	err = ctx.UpdateResource(resource, func(obj runtime.Object) error {
+	//		s := obj.(*riov1.Service)
+	//		if s.Spec.Weight == nil {
+	//			s.Spec.Weight = new(int)
+	//		}
+	//		*s.Spec.Weight = weight
+	//		s.Spec.RolloutConfig = &riov1.RolloutConfig{
+	//			Pause:           pause,
+	//			Increment:       increment,
+	//			IntervalSeconds: interval,
+	//		}
+	//		return nil
+	//	})
+	//
+	//	errs = append(errs, err)
+	//}
```

Comment:
There are still a lot of comments. @cbron are you address those?

Reply:
Yea that will all be addressed in #682